docs/content/docs/build/containers.md

```diff
@@ -142,6 +142,19 @@ container.on("disposed", () => {
 });
 ```
 
+### dirty
+
+`isDirty` flag returns a boolean value depending on the data loss in the container. The possible scenarios where the flag is set to true are:
```

Comment:
I think we should phrase this as something like "the container has operations that have not yet been acknowledged by the service. This could result in lost data." and expand from there to explain the conditions under which it will be true.